rapids-cmake/export/detail/component.cmake

```diff
@@ -51,18 +51,28 @@ function(rapids_export_component type project_name component_name export_set uni
     file(MAKE_DIRECTORY "${deps_destination}")
     install(DIRECTORY "${deps_destination}" DESTINATION "${install_location}"
             COMPONENT ${component_name})
-    install(EXPORT ${export_set}
-            FILE ${project_name}-${unique_name}-targets.cmake
-            NAMESPACE ${namespace}
-            DESTINATION "${install_location}"
-            COMPONENT ${component_name})
+    if(namespace STREQUAL "")
```

Comment:
I think this `if` / `else` is inverted. If `namespace` is empty we shouldn't specify a `NAMESPACE` argument to `install(EXPORT`

Reply:
You are correct looks like I made an error or two during conflict resolution.